src/maud/sampling.py

```diff
@@ -78,6 +80,22 @@ def sample(mi: MaudInput, output_dir: str) -> cmdstanpy.CmdStanMCMC:
     return _sample_given_config(mi, output_dir, config)
 
 
+def ppc(
```

Comment:
I don't think it's really worth having the `_given_config` function as well as the main one. The reason for doing it with `sample` and `simulate` was to avoid writing the code in `sample_given_config` twice, but in this case that's not possible so we might as well just use one function.